README.md

```diff
@@ -121,22 +106,21 @@ pnpm install
 pnpm dev
 ```
 
-> [!Note]
-> We use [Nuxt Hub](https://hub.nuxt.dev) which uses Cloudflare Pages + Cloudflare D1 databases. The database is a sqlite instance and currently the database is remote for both development and production. You can change this behavior in nuxt.config.ts.
+**Notes on Development**
 
-We use Drizzle to access the database.
+- The project uses Nuxt Hub, which runs on Cloudflare Pages with a Cloudflare D1 database.
+- The database is a SQLite instance, shared remotely in both development and production environments. To change this behavior, modify the configuration in nuxt.config.ts.
```

Comment:
This is constantly changing because NuxtHub is still in beta, so I would add a message saying that this may change and redirect to the NuxtHub docs.
For example, the database is not shared anymore, they have a [preview mode](https://hub.nuxt.com/docs/getting-started/remote-storage#production-vs-preview) that is the development db and the production one

Reply:
Even remove this section? Not really necessary